### ISSUE TITLE
Add Neurodiverse in Tech resource page

### ISSUE THREAD

**Issue by BekahHW** [Virtual-Coffee/virtualcoffee.io#814]
This issue depends on #813.

Once that issue is completed, we'll add a link for this page.

In [`developer-health`](https://github.com/Virtual-Coffee/virtualcoffee.io/tree/main/app/routes/__frontend/resources/developer-health), add a file called `neurodiverse-resources.mdx`. 

Let's use this as a space to:

1. Define the term neurodiverse
2. Write about some of the challenges neurodiverse people in tech face
3. Create a section for resources that include links that have been collected in slack.

**Comment by Nerajno** [comment]
Can I have this?

**Comment by danieltott** [comment]
@Nerajno definitely! But let's wait until #895 is merged. If you want to start now, I think starting to collect resources and write the content in the [discussion board](https://github.com/Virtual-Coffee/virtualcoffee.io/discussions) would be great!

**Comment by adiati98** [comment]
Hey @Nerajno,
I've opened a [discussion](https://github.com/orgs/Virtual-Coffee/discussions/919) for you to collect resources and/or write content for this issue while waiting for #895!

@danieltott I've added `neurodiverse.mdx` to work on this issue in the future. Hopefully #895 will be done this week!

**Comment by adiati98** [comment]
@Nerajno, just to confirm, are you still working on the content of this page?

**Comment by CBID2** [comment]
> @Nerajno, just to confirm, are you still working on the content of this page?

Hey @adiati98! :) I can help with this! :)

**Comment by Nerajno** [comment]
All yours, my hands are full right now.

**Comment by adiati98** [comment]
@CBID2 I believe @ethanfreire is working on this.

**Comment by CBID2** [comment]
> @CBID2 I believe @ethanfreire is working on this.

Ohh, I still see @Nerajno as the assigne though

**Comment by adiati98** [comment]
@CBID2 I can't assign/unassign because I'm not a maintainer of this repo.

**Comment by CBID2** [comment]
> @CBID2 I can't assign/unassign because I'm not a maintainer of this repo.

Ohh ok @adiati98. @BekahHW or @danieltott, if @ethanfreire doesn't give an update, can I work on this?
